services/github/schemas/v1/taskcluster-render-yaml-input.yml

```diff
@@ -0,0 +1,76 @@
+$schema: "/schemas/common/metaschema.json#"
+title: "Render .taskcluster.yml input"
+description: |
+  Render .taskcluster.yml for given event.
+type:         object
+properties:
+  body:
+    title: Taskcluster.yml contents
+    description: The contents of the .taskcluster.yml
+    type: string
+  organization:
+    type: string
+    description: The organization that owns the repository.
+    pattern: "^[a-zA-Z0-9_-]+$"
```

Comment:
did you get this regexp from GitHub docs?

Reply:
No, this is actually a soft-check .. couldn't find any "official regexps". 
There are some on [stackoverflow](https://stackoverflow.com/questions/59081778/rules-for-special-characters-in-github-repository-name/59082561#comment132563138_59082561) though.